recipes/tsfresh/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "tsfresh" %}
+{% set version = "0.10.1" %}
+{% set sha256 = "5866a0f8928becf9eb8a128477eb211b35ac425ef7f1cf94c277f8f97b899895" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pyscaffold
+  run:
+    - python
+    - setuptools
```

Comment:
@xhochy one minor question -- is `setuptools` needed for run or could it be dropped?

Reply:
Setuptools is required for tsfresh so that package can retrieve is version, so: no, it cannot be dropped (import tests actually fail once it's dropped).

Reply:
Ok, merging. Thanks for contributing the recipe!